TaxFoundation/itci-interactive
# Merge 'test_branch' into 'master'

'test_branch' was being used by netlify as the deploy branch. I have merged 'test_branch' into 'master' as we will be using 'master as deploy branch moving forward. The only changes made to the previous commit to master is the tax data in 'profiles.csv'. 

### Steps to test
1. clone branch
2. run 'npm install''
3. run 'npm start' 
4. go to 'localhost:8000' in browser and ensure all tax data is loading properly and subscribe links redirect to main tax foundation site.
